### DISCUSSION TITLE
[Esame 13 Febbraio 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#349] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cd832759-3426-4276-a422-c2855b8c95f9)

**Comment by Elia-Belli** [comment]
**(a)** Il primo sistema ammette soluzioni, infatti $(10,18)=2|14$, $(22,16)=2|8$ e $(9,30)=3|24$, allora possiamo semplificare il sistema e osservare che i moduli sono coprimi per ricondurci ad un cinese:

$$\begin{cases}5\equiv7(9)\\
8x\equiv 4(11)\\
3x\equiv 8(10)\end{cases}\to \begin{cases}x\equiv 5(9)\\
x\equiv 6(11)\\
x\equiv 6(10)\end{cases}$$

Procedendo col metodo di sostituzione si ottiene $x\equiv 446(990)$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a4f39f79-d88f-44f1-95a2-a54897db4227)

**Reply by pgnsamu** [reply]
Ciao, stavo provando a fare il punto b, scomponendo i moduli in fattori primi, come risultato finale mi viene x ≡ -8 (mod 30), è possibile che io abbia sbagliato qualche calcolo, oppure non era possibile utilizzare quel metodo in questa casistica?

**Reply by CiottoloMaggico** [reply]
Il punto (b) non ammette soluzione, quindi sicuramente c'è qualcosa di sbagliato anche se senza sapere cosa hai fatto non saprei dirti

**Reply by pgnsamu** [reply]
ho ricontrollato ora i passaggi, confrontandoli con un altro collega, e sono arrivato alla conclusione che sbagliavo il ragionamento nella scomposizione, grazie comunque per la risposta

**Reply by CiottoloMaggico** [reply]
di nulla
